CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java

```diff
@@ -122,6 +129,12 @@ public PluginParameters createParameters() {
         filesIncludeHeadersParam.setBooleanValue(true);
         params.addParameter(filesIncludeHeadersParam);
 
+        final PluginParameter<BooleanParameterValue> skipInvalidRows = BooleanParameterType.build(SKIP_INVALID_ROWS_ID);
+        filesIncludeHeadersParam.setName("Skip invalid rows");
+        filesIncludeHeadersParam.setDescription("True if the skip invalid rows");
+        filesIncludeHeadersParam.setBooleanValue(true);
```

Comment:
I think this is meant to be editing `skipInvalidRows` rather than `filesIncludeHeadersParam`

Reply:
Done